canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -66,6 +66,7 @@ export default class PropertiesController {
 		this.expressionRecentlyUsed = [];
 		this.expressionFieldsRecentlyUsed = [];
 		this.selectionListeners = {};
+		this.light = true; // Enable light option by default
```

Comment:
I'm not sure this is actually setting the default correctly.  Shouldn't this be a default prop in common-properties instead?